packages/safe-core-sdk/package.json

```diff
@@ -11,10 +11,16 @@
     "SDK"
   ],
   "scripts": {
+    "typechain-ethers": "typechain --target ethers-v5 --out-dir 'typechain/ethers-v5' '../../node_modules/@gnosis.pm/safe-contracts/build/contracts/**/*.json' '../../node_modules/openzeppelin-solidity/build/contracts/ERC20Mintable.json'",
```

Comment:
I'd store typechain types close to other types and they seem to be stored in `utils` folder